docs/status/wifi-performance.md

```
@@ -67,73 +67,73 @@ _Measured 2026-07-01 11:47 UTC_
-| Orange Pi R1 | ✅ | SDIO | Realtek RTL8189ES | — | 26 | 25 |
-| Orange Pi R1 | ✅ | SDIO | Realtek RTL8189ES | — | 29 | 29 |
-| Orange Pi Zero | ✅ | SDIO | XRadio XR819 | 2437 MHz @ 20 | 5 | 17 |
-| Orange Pi Zero | ✅ | USB | MediaTek MT7601 | 2437 MHz @ 20 | 12 | 13 |
-| Orange Pi Zero Plus | ✅ | SDIO | Realtek RTL8189FS | — | 33 | 27 |
-| Orange Pi Zero Plus | ✅ | SDIO | Realtek RTL8189FS | — | 35 | 26 |
-| Raspberry Pi | ✅ | SDIO | Broadcom (brcmfmac) | 2437 MHz @ 20 | 17 | 24 |
-| Rock 5B Plus | ✅ | USB | Realtek RTL8723BU 802.11b/g/n | 2437 MHz @ 20 | 13 | 17 |
-| Rock 5T | ✅ | USB | Ralink RT5572 | 5240 MHz @ 40 | 53 | 73 |
-| Tanix TX6 | ✅ | USB | Ralink RT2870/RT3070 | 2437 MHz @ 20 | 28 | 10 |
-| Udoo | ✅ | USB | Ralink RT5370 | 2437 MHz @ 20 | 34 | 18 |
+#### Wi-Fi 7 · 802.11be
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)